gui/styles/_wizard.styl

```diff
@@ -81,27 +84,83 @@ a.more-info
   color var(--slateGrey)
 
 
-.folder__selector
-  @extend $form-select select
-  width: 100%
-  font-size: 1.1em
-  line-height: 1em
-  margin 0
-  text-align left
-  text-decoration: none
+.folder__subtitle
+  margin-bottom 0
+
+.folder__config
+  margin-top 1rem
+
+  ul
+    margin-bottom 0
+    padding-inline-start: 1rem;
+
+  li + li
+    margin-top 1rem
+
+.folder__config-option__title
+  font-weight bold
+
+.folder__config-option__button
+  margin-left .5rem
+  color var(--dodgerBlue)
+
+.folder__path
+  position relative
+  padding-left 1.1rem
+
+  font-weight bold
+  font-style italic
+
+  &:before
+    @extend $icon
+    display: inline-block
+    vertical-align: middle
+    -webkit-mask-image: embedurl('./node_modules/cozy-ui/assets/icons/ui/folder.svg')
+    -webkit-mask-position: center
+    -webkit-mask-size: cover
+    mask-image: embedurl('./node_modules/cozy-ui/assets/icons/ui/folder.svg')
+    mask-position: center
+    mask-size: cover
+
+    position absolute
+    left -.5rem
+    width 1rem
+    height 1rem
+    content ''
+    background-color var(--slateGrey)
+
+    /.folder__config_error &
+      background-color var(--pomegranate)
```

Comment:
You still use the colors from the old palette?

Reply:
Yes, I haven't migrated to `cozy-bootstrap` yet and I'm using an old version of `cozy-ui`.